src/utilities.ts

```diff
@@ -261,7 +266,7 @@ export function isValidVersion(version: Version): boolean {
  * @returns Readable timestamp in the "MMDDYYYY" format.
  */
 export function getReadableTimestamp(millisTimestamp: number) {
-    return moment.utc(millisTimestamp).format("MMDDYYYY")
+    return dayjs(millisTimestamp).utc().format('MMDDYYYY')
```

Comment:
Could you add a new file to the tests folder for testing utilities like this one? We want to make sure that this is correctly outputting things without having to run a local file every time.

Reply:
Yup!